components/LayoutWithAuth0.js

```diff
@@ -1,10 +1,10 @@
+import { withAuth0 } from '@auth0/auth0-react'
 import Head from 'next/head'
 import { withRouter } from 'next/router'
-import React, { Component } from 'react'
+import { Component } from 'react'
```

Comment:
Isn't the React import still needed in this file since there is jsx?

Reply:
Good catch. Restored in https://github.com/ibi-group/otp-admin-ui/pull/30/commits/2ca33d46f23714e74e4f3c28d546279ded463ff2.